doc/Inline/C/Cookbook.swim

```diff
@@ -1222,8 +1222,9 @@ automatically so you can just concentrate on writing the functions.
   defines 2 timer events which each invoke the same callback. The first one,
   every two seconds, and the second one, every three seconds.
 
-  As of this writing, `Event.pm` is the only CPAN module that works in
-  cooperation with Inline.
+  As of this writing, Event.pm is one of a growing number of CPAN module that work in
+  cooperation with Inline. Any Alien::Build module (e.g., Alien::OpenMP) works with
```

Comment:
linking to other modules in swim can be done with `[]`, e.g. `[Event]`, `[Alien::OpenMP]`

Reply:
While you're there, please change `Event.pm` to just `[Event]` :-)